testeng/jobs/gatherCodecovMetrics.groovy

```diff
@@ -0,0 +1,50 @@
+package testeng
+
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.GENERAL_PRIVATE_JOB_SECURITY
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_MASKED_PASSWORD
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_LOG_ROTATOR
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_HIPCHAT
+
+job('gather-codecov-metrics') {
+
+    description('Gather metrics on how reliably codecov is posting statuses on pull requests')
+
+    authorization GENERAL_PRIVATE_JOB_SECURITY()
+
+    parameters {
+        stringParam('PULL_REQUEST_TIME_FRAME', '3600',
+                    'blah')
+        stringParam('CODECOV_TIME_FRAME', '1800',
+                    'blah')
```

Comment:
Shouldn't "blah" get replaced with actual descriptions for these?

Reply:
thanks. that was a placeholder until i though of something better